Libraries/LibCore/DateTime.cpp

```diff
@@ -36,6 +36,16 @@ DateTime DateTime::now()
     return from_timestamp(time(nullptr));
 }
 
+DateTime DateTime::create(unsigned year, unsigned month, unsigned day)
+{
+    DateTime dt = from_timestamp(time(nullptr));
+    dt.m_year = year;
+    dt.m_month = month;
+    dt.m_day = day;
+
+    return dt;
+}
```

Comment:
This function will create a `DateTime` with the current time as its underlying `m_timestamp`, but with its year/month/day altered. Instead, let's make it return a `DateTime` for 00:00 on the specified day.